docs/documentation/functions/aria-suite/aria-operations-networks/Update-AriaNetworksvCenterDataSourceCredentials.md

```diff
@@ -0,0 +1,131 @@
+# Update-AriaNetworksvCenterDataSourceCredentials
+
+## Synopsis
+
+Update credentials for a vCenter Server data source credentials in VMware Aria Operations for Networks.
+
+
+## Syntax
+
+Update-AriaNetworksvCenterDataSourceCredentials [-id] <String> [-collectorId] <String> [[-fqdn] <String>] [-nickname] <String> [-username] <String> [-password] <String> [<CommonParameters>]
+ [-nickname] <String> [-username] <String> [-password] <String> [<CommonParameters>]
+```
+
+## Description
+The `Update-AriaNetworksvCenterDataSourceCredentials` cmdlet allows a user to update credentials for a vCenter Server data source in VMware Aria Operations for Networks.
+
+The `Update-AriaNetworksvCenterDataSourceCredentials` cmdlet allows a user to update a previously configured vCenter Servers credentials in VMware Aria Operations for Networks.

```

Comment:
```suggestion
## Description

The `Update-AriaNetworksvCenterDataSourceCredentials` cmdlet allows a user to update credentials for a vCenter Server data source in VMware Aria Operations for Networks.

```

Reply:
Fixed in https://github.com/vmware/power-validated-solutions-for-cloud-foundation/commit/7229fe225c563ec7552b29fae42e0030b7454640